src/interface/obsidian/src/chat_view.ts

```diff
@@ -161,7 +169,10 @@ export class KhojChatView extends KhojPaneView {
 
         // Get chat history from Khoj backend and set chat input state
         let getChatHistorySucessfully = await this.getChatHistory(chatBodyEl);
-        let placeholderText = getChatHistorySucessfully ? "Message" : "Configure Khoj to enable chat";
+
+		// Check the user's os and set the modifier key accordingly
+		const modifierKey : string = Platform.isMacOS ? 'cmd' : 'ctrl';
+        let placeholderText : string = getChatHistorySucessfully ? `(${modifierKey}+↑/↓) for prev messages` : "Configure Khoj to enable chat";
```

Comment:
Should also check whether `this.userMessages.length > 1` before rendering the help tip. If there are no user messages and successful history load, then use `"Message"`.

Reply:
done, pls check.

Reply:
looks good! would be good to just address the new merge conflicts and should be good to go.

Reply:
@sabaimran  resoved the merge conflicts.